src/utils/search.js

```diff
@@ -0,0 +1,106 @@
+      h: '\u0068\u24D7\uFF48\u0125\u1E23\u1E27\u021F\u1E25\u1E29\u1E2B\u1E96\u0127\u2C68\u2C76\u0265',
+      hv: '\u0195',
+      i: '\u0069\u24D8\uFF49\u00EC\u00ED\u00EE\u0129\u012B\u012D\u00EF\u1E2F\u1EC9\u01D0\u0209\u020B\u1ECB\u012F\u1E2D\u0268\u0131',
+      j: '\u006A\u24D9\uFF4A\u0135\u01F0\u0249',
+      k: '\u006B\u24DA\uFF4B\u1E31\u01E9\u1E33\u0137\u1E35\u0199\u2C6A\uA741\uA743\uA745\uA7A3',
+      l: '\u006C\u24DB\uFF4C\u0140\u013A\u013E\u1E37\u1E39\u013C\u1E3D\u1E3B\u017F\u0142\u019A\u026B\u2C61\uA749\uA781\uA747',
+      lj: '\u01C9',
+      m: '\u006D\u24DC\uFF4D\u1E3F\u1E41\u1E43\u0271\u026F',
+      n: '\u006E\u24DD\uFF4E\u01F9\u0144\u00F1\u1E45\u0148\u1E47\u0146\u1E4B\u1E49\u019E\u0272\u0149\uA791\uA7A5',
+      nj: '\u01CC',
+      o: '\u006F\u24DE\uFF4F\u00F2\u00F3\u00F4\u1ED3\u1ED1\u1ED7\u1ED5\u00F5\u1E4D\u022D\u1E4F\u014D\u1E51\u1E53\u014F\u022F\u0231\u00F6\u022B\u1ECF\u0151\u01D2\u020D\u020F\u01A1\u1EDD\u1EDB\u1EE1\u1EDF\u1EE3\u1ECD\u1ED9\u01EB\u01ED\u00F8\u01FF\u0254\uA74B\uA74D\u0275',
+      oi: '\u01A3',
+      ou: '\u0223',
+      oo: '\uA74F',
+      p: '\u0070\u24DF\uFF50\u1E55\u1E57\u01A5\u1D7D\uA751\uA753\uA755',
+      q: '\u0071\u24E0\uFF51\u024B\uA757\uA759',
+      r: '\u0072\u24E1\uFF52\u0155\u1E59\u0159\u0211\u0213\u1E5B\u1E5D\u0157\u1E5F\u024D\u027D\uA75B\uA7A7\uA783',
+      s: '\u0073\u24E2\uFF53\u015B\u1E65\u015D\u1E61\u0161\u1E67\u1E63\u1E69\u0219\u015F\u023F\uA7A9\uA785\u1E9B',
+      ss: '\u00DF',
+      t: '\u0074\u24E3\uFF54\u1E6B\u1E97\u0165\u1E6D\u021B\u0163\u1E71\u1E6F\u0167\u01AD\u0288\u2C66\uA787',
+      tz: '\uA729',
+      u: '\u0075\u24E4\uFF55\u00F9\u00FA\u00FB\u0169\u1E79\u016B\u1E7B\u016D\u00FC\u01DC\u01D8\u01D6\u01DA\u1EE7\u016F\u0171\u01D4\u0215\u0217\u01B0\u1EEB\u1EE9\u1EEF\u1EED\u1EF1\u1EE5\u1E73\u0173\u1E77\u1E75\u0289',
+      v: '\u0076\u24E5\uFF56\u1E7D\u1E7F\u028B\uA75F\u028C',
+      vy: '\uA761',
+      w: '\u0077\u24E6\uFF57\u1E81\u1E83\u0175\u1E87\u1E85\u1E98\u1E89\u2C73',
+      x: '\u0078\u24E7\uFF58\u1E8B\u1E8D',
+      y: '\u0079\u24E8\uFF59\u1EF3\u00FD\u0177\u1EF9\u0233\u1E8F\u00FF\u1EF7\u1E99\u1EF5\u01B4\u024F\u1EFF',
+      z: '\u007A\u24E9\uFF5A\u017A\u1E91\u017C\u017E\u1E93\u1E95\u01B6\u0225\u0240\u2C6C\uA763'
+    }
+    const getKeyByValue = (object, value) => {
+      return Object.keys(object).find(key => object[key].includes(value))
+    }
+    const checkStr = Object.values(diacritics).join('')
+    str = str.split('').map(letter => (checkStr.includes(letter.toString()) ? getKeyByValue(diacritics, letter) : letter)).join('')
+    return str
+  }
+  const array = text.match(' ') ? text.split(' ') : text.split()
+  const filteredData = (ele) => { return data.filter(row => (func.find(c => replaceDiacritics(String(row[c] || '')).toLowerCase().includes(ele))))}
+  let result = filteredData(array[0])
+  for (let i = 0; i < array.length; i++) {
+    if (filteredData(array[i]).length > 0) {
+      result = filteredData(array[i])
+    }
+  }
+  return result
+}
```

Comment:
Please make sure to have end-of-file new line.

If you're using VS Code:

```
"files.trimTrailingWhitespace": true
"files.insertFinalNewline": true
```

If you're using Sublime:
```
"ensure_newline_at_eof_on_save": true
"trim_trailing_white_space_on_save": true
```

For Atom:
![default](https://user-images.githubusercontent.com/2837532/53582348-4a810200-3b4d-11e9-919c-440de72d8d1f.png)